core/src/autogluon/core/scheduler/fifo.py

```diff
@@ -399,7 +399,7 @@ def _run_reporter(self, task, task_job, reporter):
             reported_result = reporter.fetch()
             if 'traceback' in reported_result:
                 # Evaluation has failed
-                logger.exception(reported_result['traceback'])
+                logger.critical(reported_result['traceback'])
```

Comment:
Do we should use critical instead of exception? Did you check whether this has to be replaced elsewhere?

Reply:
setting to critical is to make sure the logger message won't be discard due to logger.level == `NOTSET` by default, of course we can change the default logger level, but it's just safer to ensure that the exception message has the highest logging level.